### PR TITLE
Stable9 fix 125

### PR DESCRIPTION
**Pull request by VicDeo**
Backport of #133

### CODE REVIEW

**Comment by davitol** [comment]
:+1: WFM

File: tests/itemtest.php [hunk 1]
```diff
@@ -6,44 +6,57 @@
...
+use OCA\Files_Antivirus\Item;
 
-class Test_Files_Antivirus_Item extends \OCA\Files_Antivirus\Tests\Testbase {
-	
-	/**
-	 * @var Temporary
-	 */
-	private $storage;
-	
+// mmm. IDK why autoloader fails on this class
```
Comment (PVince81) [inline tests/itemtest.php]: @VicDeo does it still fail ?

Comment (PVince81) [inline tests/itemtest.php]: ah I see, you just added a `require` to make the tests work, fine.

**Comment by PVince81** [comment]
:+1: code looks fine

**Pull request merged**